src/utils/marketxyz/fetchPoolData.ts

```diff
@@ -54,6 +53,7 @@ export const fetchPoolData = async (
   poolId: string | undefined,
   address: string | undefined,
   directory: PoolDirectoryV1,
+  ethPrice: number,

```

Comment:
Why is this a number an not a price?

Reply:
This is the current eth price

Reply:
For now that's fine but we shouldn't use numbers to represent price or token amounts it can lead to decimal precision loss